packages/cli/src/commands/versions/bump.ts

```diff
@@ -426,3 +438,35 @@ export async function bumpBackstageJsonVersion(version: string) {
     },
   );
 }
+
+async function runYarnInstall() {
+  const spinner = ora({
+    prefixText: `Running ${chalk.blue('yarn install')} to install new versions`,
+    spinner: 'arc',
+    color: 'green',
+  }).start();
+
+  const installOutput = new Array<Buffer>();
+  try {
+    await run('yarn', ['install'], {
+      env: {
+        FORCE_COLOR: 'true',
+        // We filter out all of the npm_* environment variables that are added when
+        // executing through yarn. This works around an issue where these variables
+        // incorrectly override local yarn or npm config in the project directory.
+        ...Object.fromEntries(
+          Object.entries(process.env).map(([name, value]) =>
+            name.startsWith('npm_') ? [name, undefined] : [name, value],
```

Comment:
We'd always have to print that since we have no way of knowing whether the env vars were set by the user or Yarn. If we feel it's riskier to ignore `npm_` env vars than `.npmrc` I could roll back this change, but tbh we should probably not encourage the usage of `.npmrc` then either.

Reply:
Hm ok